### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by glautervl**
- Integrating services with circleci
- Fixing services structure paths
- Adding fetch models feature
- Improving readme files

### CODE REVIEW

File: named-entity-recognition/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+RUN pip3.6 install --upgrade pip
+RUN pip3.6 install snet_cli
+RUN git clone -b named_entity_recognition https://github.com/singnet/nlp-services.git
+RUN cd nlp-services
+WORKDIR /go/src/nlp-services/named_entity_recognition
+RUN pip3.6 install -r requirements.txt
+RUN python3.6 -c "import nltk; nltk.download('all')"
+RUN sh buildproto.sh
+EXPOSE 7012-7012
+CMD ["python3.6", "run_service.py", "--daemon-conf", "config/"]
```
[the remark this answers is not in the document]
Comment (andre-senna) [reply named-entity-recognition/Dockerfile]: Not sure I understand your point. This container is not persisted, it's used only to start the service.

File: named-entity-recognition/README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+#### Output example:
+The result will be a base64 text including the chuncked sentence.
+
+After decode base64 the output will be like this:
+
+```
+$ ...
+
+```
```
[the remark this answers is not in the document]
Comment (andre-senna) [reply named-entity-recognition/README.md]: Input structure need to be well documented in README. Figuring it out by reading the command line might be misleading.

File: named-entity-recognition/services/modules/classifiers_mod.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+logger = log_config.getLogger('classifiers_mod.py')
+
+#Services Path
+current_path = os.path.dirname(os.path.realpath(__file__))
+parent_path = os.path.abspath(os.path.join(current_path, os.pardir))
+service_root_path = os.path.abspath(os.path.join(parent_path, os.pardir))
+
+# Snet Classifier
+class SnetClassifier:
```
[the remark this answers is not in the document]
Comment (andre-senna) [reply named-entity-recognition/services/modules/classifiers_mod.py]: This service may be extended to do other interesting stuff like identifying multiple references of the same entity. So having a class to contain state variables like a processing pipeline is a good thing.

**Comment by ferrouswheel** [comment]
The container is built and pushed by CI to docker hub.

Another reason for keeping commands in a single run step is that using 'rm'
as a separate command won't actually reduce the total size of the image.
Each layer stores the difference from the previous.

If you need to build Python, then you should remove the build objects and
source in the same step... Otherwise they take up unnecessary space in the
image history. This means pushing and pulling the image takes much longer,
and it can all add up.

**Comment by ferrouswheel** [comment]
If it were changed to have state or dependencies, then that would seem like
a good time to refactor into a class. As it is, I feel the use if a class
here adds confusion. Perhaps it'd be less confusing with a new class name?

**Pull request merged**